math_class.py

```
@@ -55,4 +66,66 @@ def call_model_A(wrapped_board_array, c_activated, seed):
+    if seed_id != previous_seed:
+        previous_seed = seed_id
+        seed_config = seed_configs[seed_id - 1]
+        coin_chances = seed_config['coin_chances']
+        multiplier_chances = seed_config['multiplier_chances']
+        bonus_chances = seed_config['bonus_chances']
+        C_CHANCE_PER_BLOCK = seed_config['c_chance_per_block']
+        C_CHANCE_DECREASE_SPEED = seed_config['c_chance_decrease_speed']
+        MIN_C_CHANCE_PER_BLOCK = seed_config['min_c_chance_per_block']
+
+    if is_empty_board(wrapped_board_array):
+        if seed_id <= 3:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)